src/components/PuzzleCard.vue

```diff
@@ -16,15 +16,17 @@
                 <div class="meta-tag meta-puzzleFeatures">
                     <div v-if="is3d" class="icon-3d" title="3D puzzle">3D</div>
                     <div v-if="stateCount" class="icon-multistate"></div>
-                    <StateCounter v-if="stateCount > 1" :value="stateCount" :title="`${stateCount} state switch`"/>
+                    <StateCounter v-if="+stateCount > 1" :value="stateCount" :title="`${stateCount} state switch`"/>
                 </div>  
             </div>
             <div class="meta-row">
                 <div class="meta-tag meta-user">
-                    <slot name="user-icon">
-                        <div class="user"></div>
-                    </slot>
-                    {{username}}
+                    <template v-if="madeByPlayer">
```

Comment:
I think that instead of hiding the puzzle creators if they're not players, it should be replaced by an "Eterna" username. Not having a creator is confusing and really breaks the display of the metadata on the card. Plus it doesn't make sense; not all puzzles are 3d or multistate, but all puzzles are created by someone, even if we declare that someone to be the general Eterna organization.

Reply:
This matches the current website behavior. We'll address this in the rewrite